webapp/src/components/update_post.tsx

```diff
@@ -117,10 +117,11 @@ const StyledPostText = styled(PostText)`
 `;
 
 const Separator = styled.hr`
-    && {
+    &&& {
         border: none;
         height: 1px;
         background: rgba(var(--center-channel-color-rgb), 0.61);
+        margin: 0;
```

Comment:
Though maybe this, too:

```suggestion
        margin: 12px 0;
```

<img width="107" alt="CleanShot 2022-11-14 at 10 00 27@2x" src="https://user-images.githubusercontent.com/11724372/201707072-d89678bc-cb9e-479d-8bbb-aff2ebfda2fb.png">
<img width="156" alt="CleanShot 2022-11-14 at 10 00 22@2x" src="https://user-images.githubusercontent.com/11724372/201707202-5c628c11-70fe-4988-bf20-a29508414df6.png">